src/core/devconfig.ts

```diff
@@ -93,20 +94,13 @@ export function readSourceDevConfig(): DevConfig {
     }
   };
 
-  const srcScriptPath = join(
-    quartoConfig.binPath(),
-    "..",
-    "..",
-    "scripts",
-    isWindows() ? "windows" : "common",
-  );
   return createDevConfig(
     readConfig("DENO"),
     readConfig("DENO_DOM"),
     readConfig("PANDOC"),
     readConfig("DARTSASS"),
     readConfig("ESBUILD"),
-    srcScriptPath,
+    quartoConfig.binPath(),
```

Comment:
These paths are pretty confusing to me. It seems like quartoConfig is meant to only expose paths that are in the dist folder (bin and share). If it's OK for other folders to have accessor methods, I'll add them and use them instead of these funky relative paths.

Reply:
I honestly don't remember enough about these paths to say whether a change is a good idea or likely to introduce other problems. I'd say maybe keep them as-is for now (just so we can close down v1.1) then revisit for v1.2. Does that sound okay?

Reply:
Makes sense!